index.js

```diff
@@ -17,50 +17,96 @@ const QUESTIONS = [{
 const SUBMISSION = [
 
 ]
-
+//adding middleware (are sort of inbuilt functions that give permissions to use certain expressJS features in this case) to get data from post method to a js object
+app.use(express.urlencoded({extended : true}));
+
+//function that checks if the user is existing and the password is the same:
+const authenticateUser = (email,password)=>{
+  for(user of USERS){
+    if(user.email === email && user.password === password){
```

Comment:
you can improve the time complexity of it, not that useful but definitely good for your brain :)